src/components/cc-oauth-consumer/cc-oauth-consumer.stories.js

```diff
@@ -0,0 +1,96 @@
+import { makeStory } from '../../stories/lib/make-story.js';
+import './cc-oauth-consumer.js';
+
+export default {
+  tags: ['autodocs'],
+  title: '🛠 oAuth Consumer/<cc-oauth-consumer>',
+  component: 'cc-oauth-consumer',
+};
+
+const conf = {
+  component: 'cc-oauth-consumer',
+};
+
+export const defaultStory = makeStory(conf, {
+  items: [
+    {
+      state: {
+        type: 'loaded',
+        name: 'My oAuth Consumer',
+        homePageUrl: 'https://localhost:8080/',
+        appBaseUrl: 'https://localhost:8080/',
+        description:
+          ' Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium architecto consectetur eius eum nulla obcaecati praesentium qui, voluptatum. Atque delectus enim illo, iusto modi pariatur qui repellendus? Commodi eaque, vitae?',
+        image: '',
```

Comment:
issue: can you put an image URL? Maybe the Clever Cloud logo or some fake placeholder image